source/features/separate-draft-pr-button.tsx

```diff
@@ -13,7 +13,7 @@ function init(): void | false {
 		return false;
 	}
 
-	const createPrButtonGroup = select('.hx_create-pr-button');
+	const createPrButtonGroup = select(['.hx_create-pr-button', '.timeline-comment > :last-child > .BtnGroup']);
```

Comment:
```suggestion
	const createPrButtonGroup = select([
	'.hx_create-pr-button', // GHE
	 '.timeline-comment > :last-child > .BtnGroup'
	 ]);
```

Reply:
I can add the comment, but that comment is backwards. :)

Reply:
👍 Wasn't sure. But I am sure it needs a comment 😃